docs/usage/self-hosted-configuration.md

```diff
@@ -89,20 +90,32 @@ Commas will be used as delimiter for a new filter.
 
 ```
 # DO NOT use commas inside the filter if your are using env or cli variables to configure it.
-RENOVATE_AUTODISCOVER_FILTER="/myapp/{readme.md,src/**}"
+RENOVATE_AUTODISCOVER_FILTER="/MyOrg/{my-repo,foo-repo}"
 
 # in this example you can use regex instead
-RENOVATE_AUTODISCOVER_FILTER="/myapp/(readme\.md|src/.*)/"
+RENOVATE_AUTODISCOVER_FILTER="/MyOrg\/(my|foo)-repo/"
 ```
 
 **Minimatch**:
 
+The configuration:
+
 ```json
 {
   "autodiscoverFilter": ["project/*"]
 }
 ```
 
+will match any repository in the "project" organization, while:
+
+```json
+{
+  "autodiscoverFilter": ["MyRepo/my-*"]
+}
+```
+
+will match any repo in the "MyRepo" organization that begins with "my-".
```

Comment:
```suggestion
  "autodiscoverFilter": ["my-org/*"]
}
```

will match any repository in the "my-org" organization, while:

```json
{
  "autodiscoverFilter": ["my-org/some-*"]
}
```

will match any repo in the "my-org" organization that begins with "some-".
```